duckduckgo/macos-browser
# Tespach/phishing detection pixels

Task/Issue URL: https://app.asana.com/0/1204023833050360/1208253815417548/f
Tech Design URL:
CC: https://app.asana.com/0/1204023833050360/1207699541075655/f

**Description**:
Implement EventMapping and PhishingDetectionEvents firing for pixels defined in https://app.asana.com/0/1204023833050360/1207699541075655/f

**Steps to test this PR**:
1. Start browser
2. Navigate to https://privacy-test-pages.site/security/badware/phishing.html
3. Ensure error page is thrown
4. Click through warning
5. Search app logs for "PixelKit", ensure you see something along the lines of:
```
2024-09-09 14:26:43.734612+0100 DuckDuckGo[84803:28170724] [PixelKit] 👾[Standard-Fired] m_mac_phishing_detection_error-page-shown ["appVersion": "1.105.0", "client_side_hit": "false", "pixelSource": "browser-dmg"]
2024-09-09 14:26:46.267900+0100 DuckDuckGo[84803:28170291] [PixelKit] 👾[Standard-Fired] m_mac_phishing_detection_visit-site ["appVersion": "1.105.0", "pixelSource": "browser-dmg"]
```



**Definition of Done**:


---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
